llvm/lib/Target/RISCV/RISCVVLOptimizer.cpp

```diff
@@ -930,6 +950,8 @@ bool RISCVVLOptimizer::isCandidate(const MachineInstr &MI) const {
   return true;
 }
 
+static MachineOperand One = MachineOperand::CreateImm(1);
```

Comment:
Is this a common practice to make it a static variable?
Maybe we should change `bool RISCVVLOptimizer::checkUsers` to `std::tuple<bool, const MachineOperand&> RISCVVLOptimizer::checkUsers`?

Reply:
updated.